context/app/static/js/components/Detail/visualization/Visualization/style.js

```diff
@@ -45,10 +45,11 @@ const SelectionButton = styled(Button)`
   border-radius: 3px;
 `;
 
-const EscSnackbar = styled(Snackbar)`
+const VitessceInfoSnackbar = styled(Snackbar)`
   top: ${totalHeightOffset + 10}px;
   & > div {
-    background-color: dimgray;
+    background-color: ${(props) => (props.$isWarning ? props.theme.palette.warning.main : 'dimgray')};
```

Comment:
Is there an existing palette color to use? if not, let's add this to the global palette... Might loop in @tsliaw .